### PR TITLE
Add team individuals as a codeowners

### PR DESCRIPTION
**Pull request by janisz**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1,5 @@
 * @dcos/cluster-ops-fnd
+* @makkes
+* @drewanstone
+* @janisz
+* @br-levis
```
Comment (br-lewis) [inline CODEOWNERS]:
```suggestion
* @br-lewis
```

Comment (janisz) [reply CODEOWNERS]: Thanks

File: CODEOWNERS [hunk 2]
```diff
@@ -1 +1,5 @@
 * @dcos/cluster-ops-fnd
```
Comment (makkes) [inline CODEOWNERS]: can this not be removed?

Comment (janisz) [reply CODEOWNERS]: We have a check that require person from team to approve

**Pull request merged**